src/main/java/jp/co/future/uroborosql/context/SqlContextImpl.java

```diff
@@ -75,13 +75,12 @@ public boolean contains(final Object o) {
 	}
 
 	/** where句の直後にくるANDやORを除外するための正規表現 */
-	protected static final Pattern WHERE_CLAUSE_PATTERN = Pattern
-			.compile("(?i)(?<clause>(^|\\s+)(WHERE\\s+(--.*|/\\*[^(/\\*|\\*/)]+?\\*/\\s*)*\\s*))(AND\\s+|OR\\s+)");
+	protected static final Pattern WHERE_CLAUSE_PATTERN = Pattern.compile(
```

Comment:
>  `[^(/\\*|\\*/)]`

I think this is probably not what was intended. A character class accepts only one character. Also, grouping using parentheses is not possible.

This means it is equivalent to `[^()|*/\\]`.

Reply:
Since the next `+` quantifier is non-greedy (`?`), we can probably replace it with `[\\s\\S]`, but I haven't tried it so I don't know 😅